docs/user-guide/connecting-applications.md

```diff
@@ -393,6 +393,165 @@ LoadBalancer Ingress:   a320587ffd19711e5a37606cf4a74574-1142138393.us-east-1.el
 ...
 ```
 
+## Loadbalancing multiple Services
+
+Till now we had restricted ourselves to a single Service, but Kubernetes has a resource called [Ingress](/docs/user-guide/ingress) that spans Services and loadbalances requests at [layer 7](https://en.wikipedia.org/wiki/OSI_model#Layer_7:_Application_Layer). This is useful for reasons like:
+
+* Cost saving, since the Ingress only needs one public IP, instead of 1 per Service of `Type=LoadBalancer`.
+* Centralized security configuration, instead of loading TLS certs as volumes in each Service backend.
+* Smarter routing, since the Ingress has visibility into layer 7 of the OSI model.
+
+For demonstration purposes we will deploy a second Service that simply echoes the HTTP Headers it receives:
+
+```yaml
```

Comment:
It would be so great if you could add this as an external yaml file and use the {% include code.html ...%} method documented in README.md at the root of this repo to include it. That way, users can download this file directly and/or copy its contents into their clipboard.